commands/index.js

```diff
@@ -1,12 +1,11 @@
-module.exports = function index(pkg) {
+module.exports = function index (pkg) {
   return {
     topic: pkg.topic,
     description: pkg.description,
-    help: pkg.description,
```

Comment:
To which command are you referring? the basic `heroku containers` just prints the version. What would you like added -- the sum of all the help for the other commands?